src/igvfd/schemas/assay_term.json

```diff
@@ -1,7 +1,7 @@
 {
-    "title": "Assay ontology term",
-    "description": "Schema for an ontology term.",
-    "$id": "/profiles/assay_ontology_term.json",
+    "title": "Assay term",
+    "description": "Schema for a term.",
```

Comment:
i think the description should mention assay. so it would be something like "schema for an ontological term identifying an assay."

Reply:
done